stacker/tests/test_dag.py

```diff
@@ -119,9 +122,9 @@ def test_downstream():
 
 @with_setup(start_with_graph)
 def test_all_downstreams():
-    assert dag.all_downstreams('a') == ['c', 'b', 'd']
-    assert dag.all_downstreams('b') == ['d']
-    assert dag.all_downstreams('d') == []
+    assert set(dag.all_downstreams('a')) == set(['c', 'b', 'd'])
```

Comment:
What's the reason for converting all of these to sets?  I don't think it changes the behavior really, but not sure why it's necessary.  Is this a python3'ism?

Reply:
Yeah, you are right, this change was made when I was running the test code and I do have seen `dag.all_downstreams('a')` turned into `['b', 'c', 'd']`, which failed the test. I apologize because I haven't dug deep enough to see what's going on and used `set` to mute the failure.

I'll look into this.

Reply:
Yeah, you are right, this change was made when I was running the test code and I do have seen `dag.all_downstreams('a')` turned into `['b', 'c', 'd']`, which failed the test. I apologize because I haven't dug deep enough to see what's going on and used `set` to mute the failure.

I'll look into this.